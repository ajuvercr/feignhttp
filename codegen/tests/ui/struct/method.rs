use feignhttp_codegen::feign;

// error: unknown request method marker: req
//    |  #[req]
//    |    ^^^

struct Http {}

#[feign("http://xxx")]
impl Http {
    #[req]
    pub async fn get() {}
}

fn main() {}
