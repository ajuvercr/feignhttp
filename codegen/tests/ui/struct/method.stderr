error: unknown request method marker: req
  --> tests/ui/struct/method.rs:11:7
   |
11 |     #[req]
   |       ^^^
